libraries _PLUGIN145 ITHO FAN/Itho/IthoCC1101.cpp

```diff
@@ -986,12 +988,22 @@ bool IthoCC1101::checkID(const uint8_t *id)
 	return true;
 }
 
-String IthoCC1101::getLastIDstr() {
+String IthoCC1101::getLastIDstr(bool ashex) {
 	String str;
 	for (uint8_t i=0; i<8;i++) {
-		str += String(inIthoPacket.deviceId2[i], HEX);
-		if (i<7) str += "-";
+		if (ashex) str += String(inIthoPacket.deviceId2[i], HEX);
+		else str += String(inIthoPacket.deviceId2[i]);
+		if (i<7) str += ":";
 	}
 	return str;
 }
 
+String IthoCC1101::getLastMessage2str(bool ashex) {
+    String str = "Length="+ String(inMessage2.length) + ".";
+    for (uint8_t i=0; i<inMessage2.length;i++) {
```

Comment:
Same here, please use `str.reserve(...)`

Reply:
Not 100% sure what kind of lengths can be expected here. But it seems that this function is not called at all anymore in the current version of the Itho library and could be completely removed. We don't need it in the plugin.